src/Place/Pages/PPA002Place/PlaceController.ts

```diff
@@ -0,0 +1,22 @@
+import { useNavigation } from '@react-navigation/native';
+import { NativeStackNavigationProp } from '@react-navigation/native-stack/lib/typescript/src/types';
+
+import { PlaceControllerInterface, PlacePropsInterface } from 'spelieve-common/lib/Interfaces';
+
+import { BottomTabParamList } from '@/App';
+
+export const PPA002PlaceController = ({ place_id, language }: PlacePropsInterface): PlaceControllerInterface => {
+	const navigation = useNavigation<NativeStackNavigationProp<BottomTabParamList>>();
+
+	const onCreateItineraryClicked = (placeName: string) => {
+		navigation.navigate('Place', {
+			// screen: 'IPA001ItineraryEdit',
+			params: { place_id, placeName },
+		});
```

Comment:
まだ固まってないから、TODOコメント付けておいて〜
```suggestion
		// TODO: https://github.com/Ayato-kosaka/spelieve/issues/335 Itinerary 新規作成 
		// navigation.navigate('Itinerary', {
		// 	screen: 'ItineraryTopTabNavigator',
		// 	params: { place_id, placeName },
		// });
```

Reply:
[624d7d0](https://github.com/Ayato-kosaka/spelieve/pull/326/commits/624d7d096dcd4b5bdd7e6b74fa9967b72dcf684b)